corehq/apps/users/views/mobile/users.py

```diff
@@ -1655,3 +1661,15 @@ def is_invite_valid(self):
         if hours_elapsed <= settings_obj.confirmation_link_expiry_time:
             return True
         return False
+
+@csrf_exempt
+@require_POST
+@login_or_basic_ex(allow_cc_users=True)
+def link_connectid_user(request, domain):
+    token = request.POST["token"]
+    user_info = f"{settings.CONNECTID_USERINFO_URL}"
+    user = requests.get(user_info, headers={"AUTHORIZATION": f"Bearer {token}"})
+    connectid_username = user.json().get("sub")
+    link = ConnectIDUserLink(connectid_username=connectid_username, commcare_user=request.user, domain=request.domain)
```

Comment:
I think it would be safer to do `get_or_create` to prevent duplicate records. That just makes it safer for mobile to call without causing issues.

If you want you could return 201 for created and 200 for existing - maybe useful in future if we want to debug / review the logs.